spec/requests/spree/checkout_controller_spec.rb

```diff
@@ -97,16 +97,16 @@
     context 'with logged in Bolt user' do
       let(:access_token) { 'accesstoken' }
 
-      it 'skips the service call to add addresses' do
-        expect(SolidusBolt::Accounts::AddAddressService).not_to have_received(:call)
+      it 'skips the job call to add addresses' do
+        expect(SolidusBolt::AddAddressJob).not_to have_received(:perform_later)
```

Comment:
same here as the above comment

Reply:
fixed in https://github.com/nebulab/solidus_bolt/pull/96/commits/0f5fef707383340c37526c114af66150cf238dbc